src/pages/trade/useResizablePanel.ts

```diff
@@ -0,0 +1,58 @@
+import { useEffect, useMemo, useRef, useState } from 'react';
+
+import { clamp, debounce } from 'lodash';
+
+import { timeUnits } from '@/constants/time';
+
+const PANEL_MIN_HEIGHT = 250;
+const PANEL_MAX_HEIGHT = 700;
```

Comment:
Not sure if we should move these to `constants/layouts` but we should prefix `HORIZONTAL_`

Reply:
moved to constants and prefixed